### PR TITLE
Feat: Added check for email verified before letting user write post or comment

### PR DESCRIPTION
**Pull request by lemohammed**
Changelog:

Pages: 
- disabled Write a post button
- fixed text field is editable issue
- fixed user passed as prop to comment compose in post view

Models:

- Added isEmailVerified method

Components:
- Disabled submit and write post view buttons accordingly

### CODE REVIEW

File: src/components/HomePostView.tsx [hunk 1]
```diff
@@ -70,6 +76,23 @@ const HomePostView: React.FC = () => {
                     );
                 })}
             </IonSelect>
+            <IonToast
```
Comment (RatikKapoor) [inline src/components/HomePostView.tsx:79]: 😍

**Review by RatikKapoor — APPROVED** [review]
:shipit:

**Comment by RatikKapoor** [comment]
Thanks for PR'ing straight to dev!

**Review by rjb75 — APPROVED** [review]
🎉

**Pull request merged**